test/IntegrationTests/MSTest.Acceptance.IntegrationTests/TestFilterTests.cs

```diff
@@ -33,9 +33,10 @@ public async Task DiscoverTestsWithFilter_UsingTestProperty_FilteredTests(string
         TestHostResult testHostResult = await testHost.ExecuteAsync("--filter tree=one --list-tests");
 
         testHostResult.AssertExitCodeIs(ExitCodes.Success);
-        testHostResult.AssertOutputContains("""
-The following Tests are available:
+        testHostResult.AssertOutputMatchesRegex("""
+Discovered 1 tests in assembly - .*\.dll \(net.+\|.+\)
```

Comment:
Could we update the message to have `X test(s)`?

Reply:
yes, but I find it pretty noisy, when only `1 test` is the special case here and every other number is tests (at least in english). If only we had a better way of humanizing the output.

Reply:
We could use different resource entries based on tests count :)

Reply:
done as test(s)